tasks/install.yml

```diff
@@ -1,4 +1,22 @@
 ---
+- name: Add EPEL repository
+  package:
+    name: epel-release
+    state: latest
+  tags:
+    - skip_ansible_lint
+
+# Around 2017-07-12, installing epel-release stopped being idempotent. The
```

Comment:
I'm happy to use this strategy, but I want to double-check because last time I did you and Alfredo asked me to install the epel-release package instead: https://github.com/idi-ops/ansible-ansible/pull/1#pullrequestreview-26282375

Let me know which way you prefer.